doc/dev/release-checklists.rst

```diff
@@ -144,6 +144,8 @@ See https://github.com/ceph/ceph-container/pull/2109 as an example for what to d
 After dev freeze
 ================
 
+- [ ] create branch for new release
+- [ ] remove release notes for release branch: see also 33d63c32cbf81ed81ce2185063011b98846a0d44
```

Comment:
Hrm, I guess I merged too soon.  It's already there in a different section, more detailed and with two additional steps:
```
Docs
~~~~

- [x] Remove ``doc/releases/*.rst``. This should leave behind ``doc/releases/releases.yml`` which is used for doc building purposes.
- [x] Cherry-pick 8cf9ad62949516666ad0f2c0bb7726ef68e4d666 ("doc: add releases links to toc"). There will be trivial conflicts.
- [x] Add redirect for new major release at `RTD <https://readthedocs.org/dashboard/ceph/redirects/>`_.
```

Reply:
Having this under `After dev freeze` is better because there is nowhere to remove from until the branch is created, but we need to reconcile the other two steps.

Reply:
Oops, I'll fix it.

Reply:
https://github.com/ceph/ceph/pull/56570/commits